extensions/wikia/AdEngine/js/video/player/jwplayer/jwplayerTracker.js

```diff
@@ -31,14 +35,25 @@ define('ext.wikia.adEngine.video.player.jwplayer.jwplayerTracker', [
 			return;
 		}
 
-		params.withAudio = !player.getConfig().mute;
-
 		Object.keys(trackingEventsMap).forEach(function (playerEvent) {
 			player.on(playerEvent, function(event) {
-				var errorCode;
+				var errorCode,
+					vastParams = event.tag ? vastParser.parse(event.tag) : null;
```

Comment:
As far as I understand we need to have those flags on init/load events. So we can't base on vast request beacuse it happens to late. Can't we use `params.withAudio` for that?

Also we need to make sure that value is updated

Reply:
You right. Added some code, now status is properly set on `ready` event (`init` is too soon).